lib/ex_unit/lib/ex_unit/filters.ex

```diff
@@ -48,18 +48,18 @@ defmodule ExUnit.Filters do
     end
   end
 
-  defp invalid_line_number?("0") do
-    IO.warn("invalid line number given as ExUnit filter: 0", [])
-    true
-  end
-
-  defp invalid_line_number?("-" <> _ = num) do
-    IO.warn("invalid line number given as ExUnit filter: #{num}", [])
-    true
+  defp invalid_line_number?(arg) do
+    invalid =
```

Comment:
Hi! I think you can rewrite it to this:

```
      case Integer.parse(arg) do
        {num, ""} when num > 0 -> false
        _ ->
          IO.warn("invalid line number given as ExUnit filter: #{arg}", [])
          true
      end
```

WDYT?

Reply:
Even better! Implemented in https://github.com/elixir-lang/elixir/pull/9733/commits/81b51ca45fddd215493ef4729c9438ffdceb6b61